tutorials/simulation/70_point_spread.py

```diff
@@ -116,6 +115,15 @@
     labels[i].values.fill(0.)
     labels[i].values[cent_idx] = 1.
 
+    # Print some useful information about this vertex and label
+    if 'transversetemporal' in label.name:
+        dist = label.distances_to_edge(subjects_dir=subjects_dir)[cent_idx]
+        area = label.compute_area(subjects_dir=subjects_dir)
+        # convert to equivalent circular radius
+        r = np.sqrt(area / np.pi)
+        print(f'{label.name} COM vertex is {dist * 1e3:0.1f} mm from edge '
+              f'(label area equivalent to a circle with r={r * 1e3:0.1f} mm)')
```

Comment:
> also if you find a place to document this it would be a bonus

There is an example here already, rendered as:

https://28834-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/simulation/70_point_spread.html#find-the-center-vertices-in-source-space-of-each-label

Reply:
(At some point we should probably have a tutorial for Label specifically but we don't, so this was the best place I could find)

Reply:
I meant to show the distance from label on the cortex as image but I can live without